include/exadg/fluid_structure_interaction/precice/driver.h

```diff
@@ -83,6 +83,30 @@ class Driver
   std::shared_ptr<ExaDG::preCICE::Adapter<dim, dim, VectorType>> precice;
   ExaDG::preCICE::ConfigurationParameters                        precice_parameters;
 
+  // Nomenclature
+  // time window: denotes time interval after which FSi coupling takes place
```

Comment:
FSI (capital I)

Reply:
If correct, please add "Each time window size must be the same for the fluid field and the solid field."

Reply:
I thought that the single-field solvers must hit the time window exactly? I will write another comment below ...